src/components/CreateGameDialog.vue

```diff
@@ -12,7 +12,13 @@
       <v-card-text>
         <div class="d-flex align-center">
           <v-switch v-model="isRanked" label="Ranked" data-cy="create-game-ranked-switch" />
-          <stats-scoring-dialog :show-button-text="false" data-cy="ranked-info-button" />
+          <stats-scoring-dialog :show-button-text="false" data-cy="ranked-info-button">
+            <template v-slot:button>
```

Comment:
Pref shorthand, since we're generally using it elsewhere. Could probably rummage up a linting rule for it
```suggestion
            <template #button>
```

Reply:
Forgot all about this, despite using it multiple times, hahaha.